src/tests/e2etest/E2ETest.cs

```diff
@@ -270,6 +270,13 @@ public static void AreEqualByJson(object expected, object actual)
             Assert.AreEqual(expectedJson, actualJson);
         }
 
+        public static bool AreJsonObjectsEqual(object expected, object actual)
+        {
+            string expectedJson = JsonSerializer.Serialize(expected);

```

Comment:
Can we ensure that the serialized string is always the same? For example, 2 JSON object may be the same but could be serialized to a string differently.